packages/components/teritoriNameService/ApplicationStatusCard.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { View, StyleProp, TouchableOpacity } from "react-native";
+
+import chevronRightSVG from "../../../assets/icons/chevron-right.svg";
+import {
+  gradientColorBlue,
+  gradientColorDarkerBlue,
+  gradientColorPink,
+  neutral00,
+  neutral11,
+  neutral30,
+  neutral44,
+} from "../../utils/style/colors";
+import { fontSemibold16, fontSemibold24 } from "../../utils/style/fonts";
+import { BrandText } from "../BrandText";
+import { SVG } from "../SVG";
+import { Box, BoxStyle } from "../boxes/Box";
+
+export const ApplicationStatusCard: React.FC<{
+  label: string;
+  count: any;
+  style?: StyleProp<BoxStyle>;
+  onPress: () => void;
+  isReady?: boolean;
+}> = ({ label, style, count, isReady, onPress }) => {
+  return (
+    <Box
+      notched
+      borderGradient={{
+        colors: [gradientColorBlue, gradientColorDarkerBlue, gradientColorPink],
+      }}
+      style={[
+        style,
+        {
+          flex: 1,
+          paddingVertical: 20,
```

Comment:
we should replace all margin/padding with layout.spacing

Reply:
Fixed, PR updated